src/styles/navbar.css

```diff
@@ -0,0 +1,23 @@
+.normal-navbar, .docs-navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 20px;
+}
+
+.nav-links {
+  display: flex;
+}
+
+.nav-links a {
+  margin-right: 15px;
+  text-decoration: none;
+}
+
+/* Mobile styles */
+@media (max-width: 768px) {
+  .nav-links {
+    display: none; /* Hide links */
+  }
+}
+
+  
```

Comment:
will need to convert this to `tailwindcss` for consistency with other components/pages